package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "lint": "lint-all-the-things",
     "node-test": "istanbul cover _mocha -- --recursive tests/",
+    "afternode-test": "find . -type d -empty -delete",
```

Comment:
This is to remove all the empty directories in your working copy that are left over after doing an `ember g` followed by an `ember d`.